addons/website_event_meet/controllers/community.py

```diff
@@ -34,12 +34,16 @@ def community(self, event, page=1, lang=None, **kwargs):
         :param event: event for which we display the meeting rooms
         :param lang: lang id used to perform a search
         """
+        menu = request.env["website.menu"].search([('url', '=', request.httprequest.path)])
+        view = request.env["website.event.menu"].sudo().search([('event_id', '=', event.id), ('menu_id', '=', menu.id)]).view_id
+        page = view.key if view else "website_event.template_community"
+        seo_object = request.website.get_template(page)
         return request.render(
-            "website_event_meet.event_meet",
-            self._event_meeting_rooms_get_values(event, lang=lang)
+            "website_event.template_community",
```

Comment:
Hm, do we even need an override anymore then if we can just update the template?

Reply:
Hello @reth-odoo 
Didn't got your point here, can you elaborate it.
Thanks :blush:

Reply:
Ah nevermind actually, I thought we weren't adding any values to the rendering here but we are actually